docs/release-notes/0.2/release-0.2.md

```diff
@@ -0,0 +1,61 @@
+# ML.NET 0.2 Release Notes
+
+We would like to thank the community for the engagement so far and helping us shape ML.NET.
+
+Today we are releasing ML.NET 0.2. This release focuses on addressing questions/issues, adding clustering to the list of supported machine learning tasks, enabling using data from memory to train models, easier model validation, and more.
+
+### Installation
+
+Supported platforms: Windows, MacOS, Linux (see [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more detail)
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+#### New machine learning tasks: clustering
+
+Clustering is an unsupervised learning task that groups sets of items based on their features. It identifies which items are more similar to each other than other items. This might be useful in scenarios such as organizing news articles into groups based on their topics, segmenting users based on their shopping habits, and grouping viewers based on their taste in movies. 
```

Comment:
A brief description might help give a sense of what the term means and whether it would be interesting to learn more by clicking the link. Perhaps we can adjust this once the release notes start to feel too verbose?